bootstrap/src/basis/array.ml

```diff
@@ -121,10 +121,9 @@ module Seq = struct
                                                        and type elm := T.elm =
   struct
     let to_array t =
-      let l = T.length t in
-      match l with
+      match T.length t with
       | 0 -> [||]
-      | _ -> begin
+      | l -> begin
```

Comment:
I scratched my head for a while wondering how `0` and `1` were the only cases. Finally realized this is `l` as in lookalike.

Reply:
Here's a nickel; get yourself a better font. :wink:

Reply:
It's the github default!